app/src/main/java/com/example/habitsmasher/HabitEventList.java

```diff
@@ -0,0 +1,43 @@
+package com.example.habitsmasher;
+
+import android.net.Uri;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * This is the Habit Event List Class.
+ * It's purpose is to store habit events
+ */
+public class HabitEventList extends ArrayList{
+    private ArrayList<HabitEvent> _habitEvents = new ArrayList<>();
+
+    /**
+     * Gets the list of habit events
+     * @return habitEvents: The list of habit events
+     */
+    public ArrayList<HabitEvent> getHabitEvents() {
+        return this._habitEvents;
+    }
+
+    /**
+     * Creates a new habit event and add its to the habit event list
+     * @param startDate (Date): The start date of the habit event to add
+     * @param comment (String): The comment of the habit event to add
+     * @param pictureUri (String): The URL of the picture of the habit event to add
+     */
+    public void addHabitEvent(Date startDate, String comment, Uri pictureUri) {
```

Comment:
noticed this method is not used so I'm not sure if we'll need it for later but if not I'd remove it.